<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setRules</class>
 <widget class="QWidget" name="setRules">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="CardWidget" name="frame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>531</width>
     <height>81</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Plain</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>511</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="PushButton" name="pushButton">
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PushButton" name="pushButton_2">
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PushButton" name="pushButton_3">
       <property name="text">
        <string>修改</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PrimaryPushButton" name="pushButton_4">
       <property name="text">
        <string>导入</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PrimaryPushButton" name="pushButton_5">
       <property name="text">
        <string>导出</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="CardWidget" name="frame_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>821</width>
     <height>511</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Plain</enum>
   </property>
   <widget class="TableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>781</width>
      <height>471</height>
     </rect>
    </property>
    <row/>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>协议类型</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>网络接口</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>源IP地址:端口</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>目标IP地址:端口</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>开始时间</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>结束时间</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>执行动作</string>
     </property>
    </column>
   </widget>
  </widget>
  <zorder>frame_2</zorder>
  <zorder>frame</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PrimaryPushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>TableWidget</class>
   <extends>QTableWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>CardWidget</class>
   <extends>QFrame</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
